lib/hx509/hx509.h

```diff
@@ -182,6 +182,9 @@ typedef enum {
 #define HX509_CMS_VS_ALLOW_ZERO_SIGNER			0x04
 #define HX509_CMS_VS_NO_VALIDATE			0x08
 
+/* flags from hx509_cms_verify_signed_ext */
+#define HX509_CMS_VS_VALIDATED				0x01
```

Comment:
Did you mean to make this `0x10`, or did you mean to have this flag share a value with `HX509_CMS_VS_ALLOW_DATA_OID_MISMATCH`??

Reply:
It's an output flag. Perhaps a different prefix is in order.

Reply:
Oh, right, this is an _output_ flag here.  Maybe it should have a different prefix then?